examples/text_summarization/abstractive_summarization_bertsumabs_cnndm.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>Maybe better to show the target of the first training sample?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/567/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>already fixed.</p>